atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checking =0;
+var savings = 0;
+var $checking_amount = $("#checking_amount");
+var $savings_amount = $("#savings_amount");
+var $checking_balance = $(".checking_balance");
+var $saving_balance = $(".saving_balance");
+$checking_amount.val(" ")
+savingsNewBalance = parseInt($($saving_balance.val())
+
+var checkingDeposit = function(){
+  checkingNewBalance = parseInt($checking_amount.val())
+  checking += checkingNewBalance;
+  $checking_balance.html("$"+checking);
+  if(checking === 0){
+    $checking_amount.addClass("zero")
+  }
+  else{
+    $checking_amount.removeClass("zero")
+  }
 }
 
-function checkingWithdrawal(amount){
-
+function checkingWithdrawal(){
+  checkingNewBalance = parseInt($checking_amount.val())
+  checking -= checkingNewBalance;
+  $checking_balance.html("$"+checking);
+  if(checking === 0){
+    $checking_amount.addClass("zero")
+  }
+  else {
+    $checking_amount.removeClass("zero")
+  }
 }
 
-function savingsDeposit(amount){
+function savingsDeposit(){
+  savingsNewBalance = parseInt($($saving_balance.val())
+  savings += savingsBalance;
+  $savings_balance.html("$"+savings)
+  if(savings === 0){
+    $savings_amount.addClass("zero")
+  }
+  else {
+    $savings_amount.removeClass("zero")
+  }
 
 }
 
-function savingsWithdrawal(amount){
-
+function savingsWithdrawal(){
+  savingsNewBalance = parseInt($($saving_balance.val())
+  savings -= savingsBalance;
+  $savings_balance.html("$"+savings)
+  if(savings === 0){
+    $savings_amount.addClass("zero")
+  }
+  else {
+    $savings_amount.removeClass("zero")
+  }
 }
```

Comment:
both the savingsWithdrawal and checkingWithdrawal have similar functionality, save for the value they are reading and where they are outputting the new balance. Can you think of a way to combine these two functions into one, using arguments to distinguish them?